### PR TITLE
⏱️ Readonly provider

### PR DESCRIPTION
**Pull request by Marik-D**

### CODE REVIEW

File: packages/core/src/hooks/useChainCalls.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ export function useChainCalls(calls: ChainCall[]) {
   })
 }
 
-export function useChainCall(call: ChainCall | false | undefined) {
+export function useChainCall(call: ChainCall | false | undefined | '') {
```
Comment (sz-piotr) [inline packages/core/src/hooks/useChainCalls.ts:18]:
```suggestion
type Falsy = false | 0 | "" | null | undefined 
export function useChainCall(call: ChainCall | Falsy) {
```

File: packages/core/src/providers/Providers.tsx [hunk 2]
```diff
@@ -29,3 +33,23 @@ function getLibrary(provider: any): Web3Provider {
   library.pollingInterval = 15000
   return library
 }
+
+const BACKGROUND_CHAIN_URLS = {
+  1: 'https://mainnet.infura.io/v3/62687d1a985d4508b2b7a24827551934',
```
Comment (marekkirejczyk) [inline packages/core/src/providers/Providers.tsx:38]: eee.... `id` in code?
Let's extract it to separate configuration file and use provider that by default not requires id.
Ethers have provider that work out of the box (EtherscanProvider I think).

Also:
Let's fill in urls for all test networks by default

File: packages/core/src/providers/Providers.tsx [hunk 3]
```diff
@@ -18,6 +21,7 @@ export function Providers(props: Props) {
   return (
     <Web3ReactProvider getLibrary={getLibrary}>
       <BlockNumberProvider>
+        {props.backgroundChain && <BackgroundChainActivator chainId={props.backgroundChain} />}
```
Comment (marekkirejczyk) [inline packages/core/src/providers/Providers.tsx:24]: Why Background? ReadOnly maybe?
BackgroundChainActivator -> ReadOnlyActivator

File: packages/example/src/index.tsx [hunk 4]
```diff
@@ -5,7 +5,7 @@ import { App } from './App'
 
 ReactDOM.render(
   <React.StrictMode>
-    <Providers>
+    <Providers backgroundChain={1}>
```
Comment (marekkirejczyk) [inline packages/example/src/index.tsx:8]: `<Providers readOnlyChain={1}>`?

**Review by marekkirejczyk — CHANGES_REQUESTED** [review]
See comments

**Review by marekkirejczyk — APPROVED** [review]

**Pull request merged**